.drone.star

```diff
@@ -429,6 +436,8 @@ config = {
             "runningOnOCIS": True,
             "visualTesting": True,
             "filterTags": "not @skip and not @skipOnOCIS and not @notToImplementOnOCIS",
+            "screenShots": True,
```

Comment:
Are we enabling screenshots by default? 🤔

Reply:
yes

Reply:
but why do we need to change that if it was not set before?